AndroidNativeKotlinTemplate/app/src/main/AndroidManifest.xml

```diff
@@ -48,21 +48,9 @@
         -->
         <!-- Use `android:name="com.salesforce.androidnativekotlintemplate.QrCodeEnabledLoginActivity"` when enabling log in via Salesforce UI Bridge API generated QR codes -->
 <!--        <activity-->
-<!--            android:name="com.salesforce.androidnativekotlintemplate.QrCodeEnabledLoginActivity"-->
-<!--            android:theme="@style/SalesforceSDK"-->
+<!--            android:name="com.salesforce.androidsdk.ui.LoginActivity"-->
+<!--            android:exported="true"-->
 <!--            android:launchMode="singleTask"-->
-<!--            android:exported="true">-->
-
-<!--            &lt;!&ndash; Uncomment when enabling log in via Salesforce UI Bridge API generated QR codes &ndash;&gt;-->
```

Comment:
This was still commented out here even though it also is commented on main activity where it belongs.